tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -55,12 +55,7 @@
 CLASSIFIERS = [
     'Development Status :: 5 - Production/Stable',
     'Programming Language :: Python',
-    'Programming Language :: Python :: 2',
-    'Programming Language :: Python :: 2.7',
     'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.4',
-    'Programming Language :: Python :: 3.5',
```

Comment:
Why are we removing 3.5 and 3.6 here?

Reply:
I found we keep forgetting to add newer Python versions to this `setup.py`. Simply state we are supporting for all non-EOL Python3 version is more accurate and less burdensome in the long run.